be/src/product/product.resolver.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Resolver,
+  Query,
+  Mutation,
+  Args,
+  Int,
+  ResolveField,
+  Parent,
+} from '@nestjs/graphql';
+import { ProductService } from './product.service';
+import { Product } from './entities/product.entity';
+import { CreateProductInput } from './dto/create-product.input';
+import { UpdateProductInput } from './dto/update-product.input';
+import { Item } from 'src/item/entities/item.entity';
+import { ItemService } from 'src/item/item.service';
+import { Category } from 'src/category/entities/category.entity';
+import { CategoryService } from 'src/category/category.service';
+
+@Resolver(() => Product)
+export class ProductResolver {
```

Comment:
Please add a way to add product to category (`addProductToCategory` mutation or something like that) and a way to add an item to a product as well. Functions to actually edit them (`changeItemProduct`, `changeProductCategory`) can be left for a later pr.